spec/nio/monitor_spec.rb

```diff
@@ -15,6 +15,12 @@
     expect(peer.interests).to eq(:rw)
   end
 
+  it "changes the interest set" do
+    expect(subject.interests).not_to eq(:rw)
+    subject.interests = :rw
+    expect(subject.interests).to eq(:rw)
```

Comment:
I think you need to expand this test so that it actually tries to perform some I/O operations and make calls to `select` that trigger the readiness of various `NIO::Monitors`